build.sbt

```diff
@@ -21,11 +22,11 @@ lazy val scalaLibraryNext = crossProject(JVMPlatform, JSPlatform)
     scalaModuleMimaPreviousVersion := None,
     scalacOptions ++= Seq("-deprecation", "-feature", "-Werror"),
     libraryDependencies ++= Seq(
-      "org.scalacheck" %%% "scalacheck" % "1.15.0" % Test,
+      "org.scalacheck" %%% "scalacheck" % "1.15.0" % Test
```

Comment:
If there is one place where trailing commas are useful and desirable, it is in any settings ...

Reply:
yeah, I know... this one annoyed me too.

I'm happy to try using `always` instead

Reply:
and/or bug Olaf to add a new setting that does what we want ;)

Reply:
sbt/sbt uses this:

```
trailingCommas = preserve
```